core/pom.xml

```diff
@@ -115,5 +115,17 @@
             <classifier>test</classifier>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.asynchttpclient</groupId>
+            <artifactId>async-http-client</artifactId>
+            <version>2.2.0</version>
```

Comment:
version should be defined in dependency-mgmgt section in the main pom.

Reply:
Yep, will do.